systemd/bootc-status-updated.target

```diff
@@ -2,3 +2,6 @@
 Description=Target for bootc status changes
 Documentation=man:bootc-status-updated.target(8)
 StopWhenUnneeded=true
```

Comment:
Since we're now enabling this default; I think we should probably also add here `ConditionPathExists=/run/ostree-booted`? In practice it's probably effectively the same, but the difference is that everyone who isn't using bootc will I think see "skipped as condition not met" vs "reached target bootc-status-updated" and I think the former makes more sense.

Alternatively: We could just dynamically enable this from our generator in that condition.

Reply:
:+1: good catch, updated